src/app/shared/pipe/duration.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'duration',
+})
+export class Duration implements PipeTransform {
+  transform(minutes: number) {
+    const minLimit = 0;
+    const maxLimit = 1440;
+
+    const correctNumber =
+      minutes >= minLimit && minutes <= maxLimit ? Math.floor(minutes) : 0;
+
+    const hours = Math.floor(correctNumber / 60);
+    const minutesLeft = correctNumber % 60;
+
+    const hoursCorrect = `${hours < 10 ? '0' : ''}${hours}`;
+    const minutesCorrect = `${minutesLeft < 10 ? '0' : ''}${minutesLeft}`;
```

Comment:
`10` and so on - try to avoid magic number.

Reply:
fixed